apps/table.html

```diff
@@ -415,13 +444,20 @@ <h5 class="modal-title" id="slideNameChangeModalLabel">Slide name change confirm
 
 	function initialize() {
 		slideDeleteRequests = [];
+		userCreateRequests = [];
 		const params = getUrlVars();
 		const store = new Store('../data/');
 		store.findRequest()
 			.then(function (requests) {
 				// console.log(requests);
-				if (requests) {
-					slideDeleteRequests = requests;
+				if (requests && ! requests.error) {
+					requests.forEach(function(req) {
+						if(req.type === "addUser") {
+							userCreateRequests.push(req);
+						} else {
+							slideDeleteRequests.push(req);
+						}
+					}).catch(console.error)
```

Comment:
Actually it looks like I suggested this in the wrong place...
```suggestion
					})
```

Reply:
Alright, works.